tests/providers/salesforce/hooks/test_tableau.py

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from unittest.mock import patch
+
+from airflow import configuration, models
+from airflow.providers.salesforce.hooks.tableau import TableauHook
+from airflow.utils import db
+
+
+class TestTableauHook(unittest.TestCase):
+
+    def setUp(self):
+        configuration.conf.load_test_config()
+
+        db.merge_conn(
+            models.Connection(
+                conn_id='tableau_test_password',
```

Comment:
Why this connection is required? Can't it use the `tableau_default` that was added to `utils/db.py`?